plotting/tests/test_prepare_data.py

```diff
@@ -52,41 +57,52 @@ def test_invalid_first_row(self):
             prep._check_first_row(invalid_first_row)
 
     def test_valid_first_row(self):
-        """ Test _check_first_row removes appended whitespace
-        and returns True if it matches expected_first_row """
+        """
+        Test: _check_first_row() returns True
+        When: The argument given matches PrepareData.expected_first_row,
+        despite the argument containing additional appended whitespace.
+        """
         prep = PrepareData()
         prep.expected_first_row = self.valid_first_row
         first_row = self.valid_data.split("\n")[0]
         self.assertTrue(prep._check_first_row(first_row))
 
     def test_invalid_second_row(self):
-        """ Test _check_second_row raises a RuntimeError if the given data
-        cannot be cast to an integer """
+        """
+        Test: _check_second_row() raises a RuntimeError
+        When : The argument given cannot be cast to an integer
+        """
         prep = PrepareData()
         with self.assertRaises(RuntimeError):
             prep._check_second_row("invalid")
 
     def test_valid_second_row(self):
-        """ Test _check_second_row returns a the given data as an integer
-        when the given data can be cast as such """
+        """
+        Test: _check_second_row() returns the argument given as an integer type
+        When: The argument given is a string representation of an integer """
```

Comment:
Nit: can you move the end comment `"""` to the row below?

Reply:
Good spot!